sphinx/templates/quickstart/conf.py_t

```diff
@@ -6,14 +6,11 @@
 # {{ project }} documentation build configuration file, created by
 # sphinx-quickstart on {{ now }}.
```

Comment:
I wonder if there would be more value in printing out the Sphinx version instead of the date? I don't see why the latter would be helpful, to be honest.

Reply:
IMO, both are not worthy...

Reply:
The advantage of adding the version is so you know for sure what version of Sphinx will work with a `conf.py` file (we sometimes drop support for features and options). However, I'd prefer less noise too so we can drop both.

Reply:
The generated `conf.py` might not depend on specific version of Sphinx (maybe). So it is not important.
And user can declare required version of Sphinx with `needs_sphinx` config implicitly. So I think it is not needed by default.

Reply:
I agree, both are not relevant. The less clutter the better. I would have adapted this PR, but since it has been merged already. This change will have to go to another PR. There's more to do anyway.